README.md

```diff
@@ -59,15 +58,6 @@ This repository includes files with plain SQL that can be used to recreate a dat
 
```

Comment:
![image](https://github.com/dev-mamun/vet-clinic/assets/93039359/fb7d8267-2cd9-4cff-8d43-03be793fa229)

- [ ] It would be nice if you fix your MIT link from your readme file to make your readme descriptive and professional 😊 This link is broken because you don't have an MIT file in your project 😨 Kindly put an MIT file in your repo to make your project licensed this link work 👍